community-operators/rook/rook.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,273 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
```

Comment:
Additional required fields can be found in our docs here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#required-fields-for-operatorhub